lib/parity/environment.rb

```diff
@@ -57,9 +57,9 @@ def deploy_to_heroku
     end
 
     def restore
-      if production?
+      if production? && additional_restore_arguments["--force"].nil?
```

Comment:
@jakecraige It's possible to use regex inside the square brackets too:

```ruby
additional_restore_arguments[/--force/]
```

@geoffharcourt Yes, it should be more robust. I'm thinking of doing this instead:

```ruby
!additional_restore_arguments.split(' ').include?('--force')
```

I'm not totally happy with this though as I think it would be easy to overlook the `!`